languages/portuguese/arquivos/ATIVIDADES.md

```diff
@@ -2,4 +2,72 @@
 
```

Comment:
```suggestion
> Aqui conterá uma descrição sobre submissão de atividades, avaliação de atividades e confirmação de atividades. Desde como fazer até onde fazer o gerenciamento dessas etapas.
```

Estamos seguindo esse padrão, de no início de cada página tem uma caixa de mensagem explicando resumidamente o que essa página conterá.

Reply:
Adicionado